libs/api/access-status/service/feature/src/lib/access-status.service.spec.ts

```diff
@@ -15,4 +15,20 @@ describe('AccessStatusService', () => {
   it('should be defined', () => {
     expect(service).toBeDefined();
   });
+
+  it('should be not be null',async () => {
```

Comment:
```suggestion
  it('should be not be null', async () => {
```
All of these should have spaces like this.